### ISSUE TITLE
output_issue

### ISSUE THREAD

**Issue by Amirayassermohamed** [IrohXu/lanenet-lane-detection-pytorch#23]
Hello, I am runing the pretrained model with UNet but there is an issue in output ,Can you guide me on how fix it ,I will attach the output here.
![Binary](https://user-images.githubusercontent.com/62523626/229387060-eca87866-6424-46a8-8593-ef1d67d8e89a.jpg)
![instance](https://user-images.githubusercontent.com/62523626/229387071-c09b4010-16eb-4584-80d5-48c868411294.jpg)

**Comment by IrohXu** [comment]
Are you using the `test.py` for lane marking prediction?

**Comment by Amirayassermohamed** [comment]
yes,I used test.py &changing the model from here cli_helper_test.py
ENet give good result but UNet No .

**Comment by IrohXu** [comment]
How long did you train the UNet model?

**Comment by Amirayassermohamed** [comment]
I didnot train the UNet mode ,i used your pretrained model in file log

**Comment by IrohXu** [comment]
I don't provide pretrained UNet weight. Maybe you load the wrong ENet weight into the UNet model.
